### ISSUE TITLE
Can't replicate the stated performance

### ISSUE THREAD

**Issue by Leg-end** [TJU-sjyj/AMU-Tuning#5]
Command: 
    CUDA_VISIBLE_DEVICES=0 python train.py\
            --rand_seed 2 \
            --torch_rand_seed 1\
            --exp_name test_16_shot  \
            --clip_backbone "RN50" \
            --augment_epoch 1 \
            --alpha 0.5\
            --lambda_merge 0.35\
            --train_epoch 50\
            --lr 1e-3\
            --batch_size 8\
            --shots 16\ 
            --root_path 'you root path' \
            --load_pre_feat

    CUDA_VISIBLE_DEVICES=0 python train.py\
            --rand_seed 2 \
            --torch_rand_seed 1\
            --exp_name test_1_shot  \
            --clip_backbone "RN50" \
            --augment_epoch 1 \
            --alpha 0.5\
            --lambda_merge 0.35\
            --train_epoch 50\
            --lr 1e-3\
            --batch_size 8\
            --shots 1\ 
            --root_path 'you root path' \
            --load_pre_feat
Moco-v3: ResNet-50, 1000-epoch pre-training
Device: a100

Results:
For 16-shot, the best accuracy is 69.79 at epoch 46, the stated accuracy is 70.02

For 1-shot, the best accuracy is 59.81 at epoch 1, through the entire training process, test accuracy is dropping, the final accuray is 59.10, while the stated accuray is 62.60. I also checked that the training is executing well since the final training accuracy is 100%.

The performance is quite close at 16 shot, the gap may result from randomness of training data, but the same at 1 shot is far from the stated one. Is there some special configuration for fewer shot ? But I didn't find them in the paper. 

I am not here to scrutinize your work. In fact, I find this idea really interesting and enlightment. I wanted to do further research on the logit-biase framework.  But now I am even more confused as I can't replicate findings.

Any help is greatly appreciated !

**Comment by 1108zyzy** [comment]
May I ask whether, aside from the discrepancies in these two experiments, there are any other issues?

**Comment by Leg-end** [comment]
Yes, further issues were recorded in [ISSUE 3](https://github.com/TJU-sjyj/AMU-Tuning/issues/3#issuecomment-2916671110), you can click it for detailed info

**Comment by 1108zyzy** [comment]
> [是的，第 3 期](https://github.com/TJU-sjyj/AMU-Tuning/issues/3#issuecomment-2916671110)记录了更多问题，您可以单击它查看详细信息

May I ask where you’re from? If you’re based in China, would it be possible to exchange contact details to stay in touch?
